src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -515,28 +533,38 @@ function FashionSocket({
         socket?.plugSet?.plugs.some((p) => p.plugDef.hash === plugHash)) ||
       socket?.reusablePlugItems?.some((p) => p.plugItemHash === plugHash && p.enabled));
 
+  let content = null;
+
+  if (plug) {
+    content = (
+      <PlugDef
+        onClick={handleOrnamentClick}
+        className={clsx({ [styles.missingItem]: !canSlotOrnament })}
+        plug={(plug ?? defaultPlug) as PluggableInventoryItemDefinition}
+      />
+    );
+  } else {
+    content = (
+      <PressTip tooltip={<FashSocketTooltip bucketHash={bucketHash} socket={socket} />}>
+        <div
+          role={socket && 'button'}
```

Comment:
Should this still have a tabindex so it can be selected?

Reply:
Yeah it should, I ist have missed that.